scripts/helpers/deploy-logger.js

```diff
@@ -0,0 +1,39 @@
+const flatten = require('truffle-flattener')
```

Comment:
I think you can grab this file off of `@aragon/os` as well (we will eventually move these files to another npm package...)